h2o-py/tests/testdir_algos/glrm/pyunit_glrm_PUBDEV_3454_NOFEATURE.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import print_function
+from builtins import str
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.glrm import H2OGeneralizedLowRankEstimator
+import time
+
+# I am trying to resolve a customer issue as captured in PUBDEV-3454.  Dmitry Tolstonogov (ADP)
+# said that he ran a GLRM model on a data set (which he has shared with us but would not want
+# us to put it out to public) with many categorical leves (~13000 columns in Y matrix).  Model
+# converges in ~1 hour for 76 iterations but job runs for another 6.5 hours while nothing happens.
+#  Bar indicates 10% done.
+#
+# The following test is written to duplicate this scenario and captured the stalling.  Once we
+# found the cause and fix it, this test will be used to test our results with a different dataset
+# with similar characteristic.
+
+
+def glrm_PUBDEV_3454():
+
+  feature_names = ["ooid", "emps_cnt", "client_revenue", "esdb_state", "esdb_zip", "revenue_adp", "status", "revenue_region",
+              "business_unit", "naics3"]  # column names
+  feature_types = ["enum","int","int","enum","enum","real","enum","enum","enum","int"]
+  features = ["emps_cnt", "client_revenue", "esdb_state", "esdb_zip", "revenue_adp", "status", "revenue_region",
+  "business_unit", "naics3"]
+  print("Importing user data...")
+
+  seeds = int(round(time.time()))
+
+#  seeds = 1475363366    # a good seed
+#  seeds = 1475600507    # seed with high iteration number
+  seeds = 12345
+
+  # datahex = \
+  #     h2o.upload_file(pyunit_utils.locate("/Users/wendycwong/Documents/PUBDEV_3454_GLRM/glrm_data_DTolstonogov.csv"),
+  #                     col_names=feature_names, col_types=feature_types, na_strings=["NA"])
+
+  datahex = \
+       h2o.upload_file(pyunit_utils.locate("/Users/wendycwong/Documents/PUBDEV_3454_GLRM/glrm_data_DTolstonogov.csv"),
+                       col_names=feature_names, na_strings=["NA"])
```

Comment:
I did put it in the dataset directory.  However, during pull request test, they do not want to add my dataset.  So Anmol changed the test to NOPASS for my RUNIT.  The customer does not want the dataset public.